### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by RedMser**
See tracker for details (this issue will be re-used in the future): #91521

Noteworthy changes are marked as review comments.

### CODE REVIEW

File: editor/editor_build_profile.cpp [hunk 1]
```diff
@@ -345,7 +345,7 @@ void EditorBuildProfile::_bind_methods() {
 	BIND_ENUM_CONSTANT(BUILD_OPTION_TEXT_SERVER_ADVANCED);
 	BIND_ENUM_CONSTANT(BUILD_OPTION_DYNAMIC_FONTS);
 	BIND_ENUM_CONSTANT(BUILD_OPTION_WOFF2_FONTS);
-	BIND_ENUM_CONSTANT(BUILD_OPTION_GRPAHITE_FONTS);
+	BIND_ENUM_CONSTANT(BUILD_OPTION_GRAPHITE_FONTS);
```
Comment (RedMser) [inline editor/editor_build_profile.cpp:348]: I don't believe this enum is actually exposed, despite being bound (since editor build profile is not exposed), so I do not think this is a breaking change.

Let me know if I missed something, e.g. regarding GDExtension compatibility.

File: core/string/translation.cpp [hunk 2]
```diff
@@ -702,6 +702,7 @@ void TranslationServer::setup() {
 	pseudolocalization_skip_placeholders_enabled = GLOBAL_DEF("internationalization/pseudolocalization/skip_placeholders", true);
 
 #ifdef TOOLS_ENABLED
+	ProjectSettings::get_singleton()->set_custom_property_info(PropertyInfo(Variant::STRING, "internationalization/locale/test", PROPERTY_HINT_LOCALE_ID, ""));
```
Comment (AThousandShips) [inline core/string/translation.cpp:705]: This should be in a separate PR as it is a functional change unlike everything else

File: modules/lightmapper_rd/lightmapper_rd.cpp [hunk 3]
```diff
@@ -1694,7 +1694,7 @@ LightmapperRD::BakeError LightmapperRD::bake(BakeQuality p_quality, bool p_use_d
 		light_probe_buffer = rd->storage_buffer_create(sizeof(float) * 4 * 9 * probe_positions.size());
 
 		if (p_step_function) {
-			p_step_function(0.7, RTR("Baking lightprobes"), p_bake_userdata, true);
+			p_step_function(0.7, RTR("Baking light probes"), p_bake_userdata, true);
```
Comment (RedMser) [inline modules/lightmapper_rd/lightmapper_rd.cpp:1697]: This changes a translation string.

File: editor/editor_build_profile.cpp (hunk 1 above)
Comment (AThousandShips) [inline editor/editor_build_profile.cpp:348]: This breaks compatibility and can't be done here

Comment (AThousandShips) [reply editor/editor_build_profile.cpp:348]: Let's see what the docgen says, and the extension

File: core/string/translation.cpp (hunk 2 above)
Comment (RedMser) [reply core/string/translation.cpp:705]: OK. It might make sense to batch them just like typos, so we have one "fix X typos" PR and another "fix X UI papercuts" like propertyinfo and basic/advanced project settings flags, which can be reviewed separately.

File: editor/editor_build_profile.cpp (hunk 1 above)
Comment (RedMser) [reply editor/editor_build_profile.cpp:348]: Checks seem to pass successfully, so it should be fine?

**Review by Mickeon — APPROVED** [review]
Should be fine

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!
